app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask, jsonify
+import json
+import pdb
+
+# instantiate the app
+app = Flask(__name__)
+app.config.from_object(__name__)
+
+@app.route('/emoji', methods=['GET'])
```

Comment:
this is the correct pluralization of emoji.

Reply:
😝